creation/disks-and-volumes.md

```diff
@@ -431,7 +431,7 @@ those PVCs.
 
 In order to take advantage of the DataVolume volume source on a VM or
 VMI, the **DataVolumes** feature gate must be enabled in the
-**kubevirt-config** config map before KubeVirt is installed. CDI must
+**kubevirt** custom resource before KubeVirt is installed. CDI must
```

Comment:
I'm unsure if this is still true

Reply:
Let's check the virt-controller and virt-api logic explicitly, but I think it is no longer true.